HelpSource/Classes/NodeProxy.schelp

```diff
@@ -189,6 +189,23 @@ Ndef(\maus, { LFNoise0.kr(1 ! 8) + 1 }); // now 8 parallel channels of audio are
 
 
 
+subsection::Reserved parameters
+Three parameters are automatically specified if they don't exist in a given UGen function. You can override their use: code::[\out, \gate, \fadeTime]::
```

Comment:
which three parameters?

Reply:
\out, \gate, \fadeTime. this reads OK to me.

Reply:
looks totally ok, I just missed that github code is not wrapping:

<img width="1130" alt="screen shot 2018-07-07 at 17 02 59" src="https://user-images.githubusercontent.com/75468/42412187-aa3f8518-8207-11e8-8009-55b61c958b9f.png">